integration-tests/deployment/ccip/test_helpers.go

```diff
@@ -232,3 +248,68 @@ func AddLanesForAll(e deployment.Environment, state CCIPOnChainState) error {
 	}
 	return nil
 }
+
+const (
+	// nolint:lll
+	// https://github.com/smartcontractkit/chainlink/blob/a348b98e90527520049c580000a86fb8ceff7fa7/contracts/src/v0.8/tests/MockV3Aggregator.sol#L76-L76
+	MockLinkAggregatorDescription = "v0.8/tests/MockV3Aggregator.sol"
```

Comment:
Having relative file paths like this in tests seems smelly. What are we trying to achieve with this setup?

Reply:
I'll explain more in the comments above, I added the link to the `description` function above it as well.